draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -465,6 +474,35 @@ even when also exporting the expanded key. ML-DSA seed extraction can be
 implemented by including the random seed xi generated at line 1 of Algorithm 1
 ML-DSA.KeyGen in the returned output.
 
+# Pairwise Consistency Testing
+
+This section specifies RECOMMENDED pairwise consistency testing
+procedures for ML-DSA implementations as defined in {{FIPS204}}.
+These procedures align with the Implementation Guidance for {{FIPS140}}
+and the Cryptographic Module Validation Program Section
+10.3.A. The tests help verify correct implementation of key
+generation, signing, and verification operations.
+
+## Pairwise Consistency Test Requirements
+
+Implementations generating key pairs for use in certificates
+SHOULD verify these properties before accepting a key pair as valid:
```

Comment:
"SHOULD" CAs generally verify the ML-DSA public key before issuing the cert?

Reply:
Yes, I added some text here to describe what they might like to do, it essentially falls in line exactly with FIPS 204 section 3.6 additional requirements. i.e., check the public key is of the correct size given the parameter set. [def88ec](https://github.com/lamps-wg/dilithium-certificates/pull/88/commits/def88ec82eab0e26964913430cdc9edbf736eb9b)